resources.go

```diff
@@ -509,8 +542,16 @@ func Provider() tfbridge.ProviderInfo {
 			"google_runtimeconfig_config":   {Tok: gcpResource(gcpRuntimeConfig, "Config")},
 			"google_runtimeconfig_variable": {Tok: gcpResource(gcpRuntimeConfig, "Variavble")},
```

Comment:
Not your code, but can you fix this typo while you're at it:
```suggestion
			"google_runtimeconfig_variable": {Tok: gcpResource(gcpRuntimeConfig, "Variable")},
```

Technically a breaking change, so might as well do it while we're at it?